push-notifications/README.md

```diff
@@ -303,7 +305,7 @@ checkPermissions() => Promise<PermissionStatus>
 
 Check permission to receive push notifications.
 
-On Android the status is always granted because you can always
+On Android 12 and below the status is always granted because you can always
```

Comment:
this texts comes from the types, should be edited there and then run `npm run build` to propagate it to the README
https://github.com/ionic-team/capacitor-plugins/blob/main/push-notifications/src/definitions.ts#L93-L117

Reply:
Ah, thanks for catching that - I was in markdown mode going through and changing everything.